ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/EventsTable.scala

```diff
@@ -59,14 +60,16 @@ private[events] trait EventsTable {
     private def instantToTimestamp(t: Instant): Timestamp =
       Timestamp(seconds = t.getEpochSecond, nanos = t.getNano)
 
-    def toFlatTransaction(events: List[Entry[Event]]): Option[GetFlatTransactionResponse] = {
+    private def flatTransaction[R](makeResponse: ApiTransaction => R)(
+        events: Seq[Entry[Event]],
+    ): Option[R] =
       events.headOption.flatMap { first =>
-        val flatEvents =
-          TransactionConversion.removeTransient(events.iterator.map(_.event).toVector)
-        if (flatEvents.nonEmpty || first.commandId.nonEmpty)
-          Some(
-            GetFlatTransactionResponse(
-              transaction = Some(
+        {
```

Comment:
This seems totally unnnnnnnnnnnecessary.

Reply:
Indeed, probably a leftover while moving code around, thanks. Addressed by https://github.com/digital-asset/daml/pull/5187/commits/d3d7540935cb4a71dead935e287d3145a3ef4233